### ISSUE TITLE
The Info event does not always return the symbol that is drawn on top

### ISSUE THREAD

**Issue by pauldendulk** [pauldendulk/Mapsui#191]
The order of drawing and the order of searching for the tapped symbol should be the same.

**Comment by Sebastian1989101** [comment]
Is there already a target date, when this will be fixed?

**Comment by pauldendulk** [comment]
I hope to get round this next week. You could also try to take a look yourself. I assume you just need to remove this line from InfoHelper.cs.
```
var feature = features.OrderBy(f => f.Geometry.GetBoundingBox().GetCentroid().Distance(worldPosition)).FirstOrDefault();
```
It needs to be tested though. Preferably in a unit this in FeatureInfoTests.cs.

**Comment by pauldendulk** [comment]
Fix published in 1.0.8
